app/assets/stylesheets/application.css.scss

```diff
@@ -41,6 +41,11 @@ ul {
     margin: 0;
 }
 
+.stat-bubble {
+  background:rgba(0,0,0,0.5);
```

Comment:
Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space